ServerCore/Pages/Resources/Default/HomePartial.cshtml

```diff
@@ -15,5 +19,9 @@
         {
             <p><b>Since you're an admin:</b> Either set the HomePartial field of your event to match a folder in the Pages\Resources directory, or place raw HTML into the HomeContent field of your event.</p>
         }
+        if (eventPage?.EventRole == ModelBases.EventRole.author)
+        {
+            <p><b>Since you're an author:</b> Take a look at <a src="https://github.com/PuzzleServer/mainpuzzleserver/wiki/Authors:-How-to-use-the-site">Authors: How to use the site</a> wiki for more information on how to use this site as an author.</p>
```

Comment:
For correctness, links use `href`, not `src`.  Also, I recommend all external links add a `target="_blank"` attribute as well so they always open in a new tab by default.

Reply:
Done! That Href allowed me to remove the styling I added as well